docs/DeveloperGuide.md

```diff
@@ -257,17 +257,32 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* works in the housing management team for RC4 with several other co-workers
+* has a need to manage a significant number of residents in RC4
+* is responsible for performing a wide variety of tasks including liasing with students/staff
+* requires quick access to contact details and other relevant resident information 
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: 
+
+* manage contacts faster than a typical mouse/GUI driven app
+* requires less technical knowledge to perform complex tasks
+* easier on the eyes, as compared to compressed rows of data on Excel
 
 
 ### User stories
 
+Our user stories have been packaged with the relevant functionalities that we will/have implemented.
```

Comment:
Can we change the last part of the sentence to "that we will implement/have implemented"?